test_suite.sh

```diff
@@ -132,56 +132,73 @@ else
 fi
 
 # Configure ReFrame, see https://www.eessi.io/docs/test-suite/installation-configuration
-export RFM_CONFIG_FILES=$TOPDIR/reframe_config_bot.py
-export RFM_CONFIG_FILE_TEMPLATE=$TOPDIR/reframe_config_bot.py.tmpl
+# RFM_CONFIG_FILES _has_ to be set by the site hosting the bot, so that it knows where to find the ReFrame
+# config file that matches the bot config. See https://gitlab.com/eessi/support/-/issues/114#note_2293660921
+if [ -z "$RFM_CONFIG_FILES" ]; then
+    err_msg = "Please set RFM_CONFIG_FILES in the environment of this bot instance to point to a valid"
+    err_msg = "${err_msg} ReFrame configuration file that matches the bot config."
+    err_msg = "${err_msg} For more information, see https://gitlab.com/eessi/support/-/issues/114#note_2293660921"
+    fatal_error "${err_msg}"
```

Comment:
```suggestion
    err_msg="Please set RFM_CONFIG_FILES in the environment of this bot instance to point to a valid"
    err_msg="${err_msg} ReFrame configuration file that matches the bot config."
    err_msg="${err_msg} For more information, see https://gitlab.com/eessi/support/-/issues/114#note_2293660921"
    fatal_error "${err_msg}"
```

This will resolve the error that I'm seeing but this is not an explanation on why it cannot find $RFM_CONFIG_FILES did you put it in your bachrc @casparvl?